src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -386,5 +390,18 @@ internal bool TryGetByName(string name, out RecalcFormulaInfo info)
             info = null;
             return false;
         }
+
+        public bool TryGetVariableType(string name, out FormulaType type)
+        {
+            type = default;
+            var symbols = CreateResolverInternal();

```

Comment:
>var symbols = CreateResolverInternal(); [](http://example.com/codeflow?start=12&length=39)

This should have more symmetry with above methods like TryGetByName. 

We can probably just check against _symbolTable directly rather than call CreateResolverInternal() 

This means we're just operating on the current variables added here. 
If we want something more general, it should be on Engine base class.

Reply:
updated.